app/config/config/schema.js

```diff
@@ -55,6 +55,7 @@ const configSchema = Joi.object({
       .forbidden()
       .messages({ 'any.unknown': 'The `sever.registration_locked` option is moved to `service.invite_code`' }),
     cors: Joi.alternatives().try(Joi.boolean(), Joi.keyArray()).default(false),
+    max_file_upload: Joi.string().required(),
```

Comment:
what about a default value here, otherwise this option is required to be set manually which would be breaking.

Reply:
Oh, you are right. I will set it to 1mb